### PR TITLE
Feature/styleguide

### PR DESCRIPTION
**Pull request by katlazarova**

### CODE REVIEW

File: web/themes/custom/pineapple/sass/base/_buttons.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+.js-form-submit.form-submit {
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/sass/base/_buttons.scss]: As discussed use the button class and not drupal generated classes i.e `.js-form-submit`

File: web/themes/custom/pineapple/sass/base/_buttons.scss [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+  &#edit-submit--2 {
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/sass/base/_buttons.scss]: Same as above, create button variants and do not use id's

File: web/themes/custom/pineapple/sass/base/forms/_form-items.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.form-select {
+  border: 1px solid #ccc;
+  background: #fff;
+  height: 25px;
+}
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/sass/base/forms/_form-items.scss]: Delete this entire file.

File: web/themes/custom/pineapple/sass/base/_buttons.scss (hunk 1 above)
Comment (katlazarova) [reply web/themes/custom/pineapple/sass/base/_buttons.scss]: done

File: web/themes/custom/pineapple/sass/base/_buttons.scss (hunk 2 above)
Comment (katlazarova) [reply web/themes/custom/pineapple/sass/base/_buttons.scss]: done

File: web/themes/custom/pineapple/sass/base/forms/_form-items.scss (hunk 3 above)
Comment (katlazarova) [reply web/themes/custom/pineapple/sass/base/forms/_form-items.scss]: done.

**Pull request merged**